opencage_geocoder.go

```diff
@@ -68,7 +77,12 @@ func (g *OpenCageGeocoder) Request(url string) ([]byte, error) {
 // if one occurs during the geocoding request.
 func (g *OpenCageGeocoder) Geocode(query string) (*Point, error) {
 	url_safe_query := url.QueryEscape(query)
-	data, err := g.Request(fmt.Sprintf("?q=%s&pretty=1", url_safe_query))
+	key := ""
+	if (OpenCageAPIKey != "") {
+		key = "key="+OpenCageAPIKey+"&"
+	}
+
+	data, err := g.Request(fmt.Sprintf("?%sq=%s&pretty=1", key, url_safe_query))
```

Comment:
The OpenCage Geocoder API seems to suggest using a `key` URL parameter [as well](http://geocoder.opencagedata.com/api.html).

I think we want `key=%s&q=%s&pretty=1` here.